releasenotes/notes/float-context-1a05941c69c21bdd.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Gate-parameter contexts now permit ``float`` and ``angle`` types to interact via multiplication
```

Comment:
Should you mention this is only if strict mode is disabled?

Reply:
Done in e4b95ec.